docs/guides/react-native.md

```diff
@@ -9,32 +9,188 @@ a host device.
-```bash
-npm i --save-dev react-native-mock
+While a React Native adapter is [in discussion](https://github.com/airbnb/enzyme/issues/1436),
+a standard adapter may be used, such as "enzyme-adapter-react-16":
+
+```
+import Adapter from "enzyme-adapter-react-16";
+
+Enzyme.configure({ adapter: new Adapter() });
+```
+
+## Loading an emulated DOM with JSDOM
+
+To use enzyme's `deep` until a React Native adapter exists, an emulated DOM must be loaded.
+
+While some have had success with [react-native-mock-renderer](https://github.com/Root-App/react-native-mock-render),
+the recommended approach is to use [https://github.com/tmpvar/jsdom](JSDOM),
+as documented for enzyme at the [JSDOM](https://airbnb.io/enzyme/docs/guides/jsdom.html) documentation page.
+
+JSDOM will allow all of the `enzyme` behavior you would expect, and it will result in reasonable snapshots if
+used in a Jest snapshot test.
+
+## Using enzyme's find when lacking className props
+
+It is worth noting that React Native allows for a [testID](https://facebook.github.io/react-native/docs/view#testid)
+prop, that can be used a selector similar to `className` in standard React:
+
+```javascript
+    <View key={key} style={styles.todo} testID="todo-item">
+      <Text testID="todo-title" style={styles.title}>{todo.title}</Text>
+    </View>
 ```
 
-Requiring or importing the `/mock` entry file of this project will input the mock `react-native`
-export into the require cache, so that your application uses the mock instead.
+```javascript
+  expect(
+    wrapper.findWhere(node => node.prop("testID") === "todo-item"),
+  ).toExist();
+```
+
+## Example configuration for Jest
+
+To perform the necessary configuration in your testing framework, it is recommended to use a setup script,
+such as with Jest's `setupTestFrameworkScriptFile` setting.
 
-If you are using a test runner such as mocha, this means that you can use the `--require` flag
-before you run your test suite, and enzyme should "just work":
+Create or update a `jest.config.js` file at the root of your project to include the `setupTestFrameworkScriptFile` setting:
 
+```javascript
+// jest.config.js
 
-### Mocha CLI
+module.exports = {
+  // Load setup-tests.js before test execution
+  setupTestFrameworkScriptFile: "<rootDir>setup-tests.js",
 
-```bash
-mocha --require react-native-mock/mock --recursive path/to/test/dir
+  // ...
+};
 ```
 
-### In Code
+Then create or update the file specified in `setupTestFrameworkScriptFile`, in this case `setup-tests.js` in the project root:
+
+```javascript
+// setup-tests.js
+
+import "react-native";
+import "jest-enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import Enzyme from "enzyme";
+
+/**
+ * Set up DOM in node.js environment for Enzyme to mount to
+ */
+const { JSDOM } = require("jsdom");
+
+const jsdom = new JSDOM("<!doctype html><html><body></body></html>");
+const { window } = jsdom;
+
+function copyProps(src, target) {
+  const props = Object.getOwnPropertyNames(src)
```

Comment:
is `Object.getOwnPropertyDescriptors` not an option?

Reply:
Let's update to use it; it's cleaner anyways.

Reply:
Do we want to support es5 / IE?

I updated for now to:

```
function copyProps(src, target) {
  Object.defineProperties(target, {
    ...Object.getOwnPropertyDescriptors(src),
    ...Object.getOwnPropertyDescriptors(target)
  });
}
```

I think that does what we want cleanly, but VSCode is complaining:

```
[ts] Property 'getOwnPropertyDescriptors' does not exist on type 'ObjectConstructor'. Did you mean 'getOwnPropertyDescriptor'?
lib.es5.d.ts(172, 5): 'getOwnPropertyDescriptor' is declared here.
```

Because Object.getOwnPropertyDescriptors isn't in the es5 spec and unsupported also for IE:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptors

Reply:
Let me know whether we should keep as is to support es5 / IE, or whether the updated code I have above with `getOwnPropertyDescriptors` is preferred.

Reply:
gOPDs is polyfillable, so it should work just fine. https://www.npmjs.com/package/object.getownpropertydescriptors

The spread can be transpiled to Object.assign which is also polyfillable https://www.npmjs.com/package/object.assign

so i think that update works great

Reply:
Sounds good, I updated copyProps here and in two other places in the codebase:

docs/guides/jsdom.md
packages/enzyme-example-mocha/test/.setup.js